### PR TITLE
Showing proper message for down server

### PR DESCRIPTION
**Pull request by HarisShahidHameed**
Trello link : https://trello.com/c/XTcdnoGI/70-show-proper-error-if-backend-server-is-down

### CODE REVIEW

File: src/api/index.js [hunk 1]
```diff
@@ -71,6 +71,15 @@ const userAboutInfo = (data, resolve, reject) => {
     .then(res => resolve(res))
     .catch(err => reject(err));
 };
+
+const serverDownErrorMessage = (error) => {
+  debugger
+  if (error === 'Failed to fetch') {
```
Comment (abdulhaqshah) [inline src/api/index.js:76]: Can we change the error message from backend? if yes then no need for frontend changes. 

Message should be: "Something went wrong try again later"

**Review by abdulhaqshah — CHANGES_REQUESTED** [review]

File: src/api/index.js (hunk 1 above)
Comment (HarisShahidHameed) [reply src/api/index.js:76]: Changed

File: src/api/index.js [hunk 2]
```diff
@@ -71,6 +71,15 @@ const userAboutInfo = (data, resolve, reject) => {
     .then(res => resolve(res))
     .catch(err => reject(err));
 };
+
+const serverDownErrorMessage = (error) => {
+  debugger
```
Comment (abdulhaqshah) [inline src/api/index.js]: what is this

File: src/api/index.js (hunk 1 above)
Comment (abdulhaqshah) [inline src/api/index.js:76]: should match with status not string

File: src/api/index.js [hunk 3]
```diff
@@ -71,6 +71,15 @@ const userAboutInfo = (data, resolve, reject) => {
     .then(res => resolve(res))
     .catch(err => reject(err));
 };
+
+const serverDownErrorMessage = (error) => {
```
Comment (abdulhaqshah) [inline src/api/index.js]: getErrorMessage only

**Review by abdulhaqshah — CHANGES_REQUESTED** [review]

File: src/api/index.js (hunk 2 above)
Comment (HarisShahidHameed) [reply src/api/index.js]: Removed.

File: src/api/index.js (hunk 1 above)
Comment (HarisShahidHameed) [reply src/api/index.js:76]: Can only match with string because it only contains string

File: src/api/index.js (hunk 3 above)
Comment (HarisShahidHameed) [reply src/api/index.js]: Done

**Review by abdulhaqshah — APPROVED** [review]

**Pull request merged**